region/maridia/outer/Mt. Everest.json

```diff
@@ -1398,6 +1399,16 @@
       "requires": [
         "h_canNavigateUnderwater",
         "canTrickyUseFrozenEnemies",
+        {"or": [
+          "canInsaneJump",
+          "Spazer",
+          "Wave",
+          "Plasma",
+          {"and": [
+            "canTrickyJump",
+            {"ammo": {"type": "Super", "count": 1}}
+          ]}
```

Comment:
It's really hard for me to know if this is reasonable, as it's hard not to know the room is coming. Maybe `canPrepareForNextRoom` makes more sense than a tricky jump?
```suggestion
          ]},
          {"and": [
            "canTrickyJump",
            "HiJump"
          ]}
```